.github/workflows/main.yml

```diff
@@ -31,9 +31,27 @@ jobs:
           node-version: '12.x' # TODO: match nvmrc version
       - name: Install dependencies
         run: yarn --frozen-lockfile
+      - name: Setup sonar scanner
+        uses: warchant/setup-sonar-scanner@v1
```

Comment:
Is there a particular reason why this is between installing dependencies and linting?

Reply:
nop, we just need to have the plugin before run the `sonarscanner` step.